src/social-web/routes/feed/stage.tsx

```diff
@@ -125,18 +131,44 @@ export default function FeedStage( { onSelectItem }: FeedStageProps ) {
 		[ view.fields, updateView ]
 	);
 
+	// Extract ap_object_type filter from view.filters
+	const apObjectTypeFilter = useMemo( () => {
+		const typeFilter = view.filters?.find( ( f ) => f.field === 'ap_object_type' );
+		return typeFilter?.value as number[] | undefined;
+	}, [ view.filters ] );
+
 	const { feed, isResolving, totalItems, totalPages } = useFeed( {
 		perPage: view.perPage || 20,
 		page: view.page || 1,
 		orderBy: view.sort?.field || 'date',
 		order: view.sort?.direction || 'desc',
 		search: view.search || '',
 		userId: activeActorId,
+		apObjectType: apObjectTypeFilter,
```

Comment:
```suggestion
		filters: view.filters,
```

Should we pass all `view.filters` here and let `useFeed()` dissect them?